deployment/pom.xml

```diff
@@ -4,8 +4,7 @@
   <parent>
     <groupId>io.quarkiverse.logging.splunk</groupId>
     <artifactId>quarkus-logging-splunk-parent</artifactId>
-    <version>1.1.0-SNAPSHOT
-    </version>
+    <version>2.0-SNAPSHOT</version>
```

Comment:
As discussed, let's try semver (new major version of extension each time there's an incompatibility with Quarkus)